### ISSUE TITLE
Re-draw and translate several images

### ISSUE THREAD

**Issue by trgiangdo** [trgiangdo/ml-glossary-vn#15]
- [ ] autoencoder.png
- [ ] backprop_visually.png
- [ ] calculus_slope_intro.png
- [ ] cnn_filter_output.png
- [ ] cnn.jpg
- [ ] cross_entropy.png
- [ ] decision_tree.png
- [ ] dropout_net.png
- [ ] dropout.png
- [ ] early_stopping.png
- [ ] gan.png
- [ ] gradient_accumulator.png
- [ ] gru_structure.png
- [ ] learned_regression_line.png
- [ ] learned_regression_line1.png
- [ ] learned_regression_line2.png
- [ ] learned_regression_line3.png
- [ ] learned_regression_line4.png
- [ ] linear_regression_line_intro.png
- [ ] linear_regression_training_cost.png
- [ ] logistic_regression_binary_decision_boundary.png
- [ ] logistic_regression_exam_scores_scatter.png
- [ ] logistic_regression_final_decision_boundary.png
- [ ] logistic_regression_loss_history.png
- [ ] logistic_regression_scatter_w_decision_boundary.png
- [ ] logistic_regression_sigmoid_w_threshold.png
- [ ] lstm_structure.png
- [ ] maxpool.png
- [ ] mlp.png
- [ ] multiple_regression_error_history.png
- [ ] neural_network_simple.png
- [ ] neural_network_w_matrices.png
- [ ] neuron.png
- [ ] regularization-dropout.png
- [ ] rnn_layer.png
- [ ] rnn.png
- [ ] simple_nn_diagram_zo_zh_defined.png
- [ ] vae.png
- [ ] y1andy2_logistic_function.png

**Comment by dungchitobe9** [comment]
@trgiangdo em tag anh vào xong nhận assign đúng k ạ

**Comment by trgiangdo** [comment]
@dungpc-69 Cái này là Issue khá to, anh sẽ chia phần này ra sau. Em tạm thời cứ làm 1 phần dịch tiếng Anh sang tiếng Việt trong docs trước đi, 1 đoạn thôi cũng được để quen dần cách dùng Git.

Sau anh sẽ chia Issue và giao cho từng bạn sau.
